src/index.js

```diff
@@ -14,6 +14,23 @@ const getDefaultPayload = () => ({
   },
 });
 
+const getSessionHistory = () => {
+  const history = localStorage.getItem('history');
+
+  return history ? JSON.parse(history) : [];
```

Comment:
should probably put `JSON.parse` in a try catch in case localStorage is corrupted or tampered with

Reply:
and check if it's an array